configure.ac

```diff
@@ -32,16 +32,12 @@
 #
 # Declare autoconf version requirements
 #
-AC_PREREQ([2.68])
+AC_PREREQ([2.71])
```

Comment:
Are the changes in this file intentional/necessary?

Reply:
Sorry, those are unintentional. I'm on Ubuntu 22.04 now, and I think that I might have accidentally run a command to update the autoconf file to the latest version. I've just updated the pull request to remove the changes to this file.